snapcraft.yaml

```diff
@@ -19,6 +19,16 @@ apps:
     restart-condition: always
 
 parts:
+  curtin:
+    plugin: python
+    source-type: git
+    source: git://git.launchpad.net/curtin
+    source-branch: ubuntu/devel
+    requirements: requirements.txt
+    organize:
+      'lib/python*/site-packages/usr/lib/curtin': 'usr/lib/'
+    stage: [ "*", "-lib/python*/site-packages/_yaml.*.so" ]
```

Comment:
I wonder why this looks like a syntax error, by oh well.

Reply:
yaml be yaml